<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSingleUpgradeMainControl</class>
 <widget class="QWidget" name="WidgetSingleUpgradeMainControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>453</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>532</width>
       <height>366</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>532</width>
       <height>366</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FAFDFF;
border-color:#FAFDFF;
border-radius: 4px;
color:#2D3440;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>56</number>
      </property>
      <property name="rightMargin">
       <number>56</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>62</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
font-size: 22px;
font-weight: 500;
color: #2D3440;</string>
        </property>
        <property name="text">
         <string>升级主控板</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QRadioButton" name="btnRadioSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>43</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton
{
	spacing: 6px;
	border:none;
color: #2D3440;
font-size: 16px;
font-weight: 350;
}
QRadioButton::indicator
{
	image: url(:/image/images2/icon_single_radioUnCheck.png);
}
QRadioButton::indicator:unchecked 
{
	image: url(:/image/images2/icon_single_radioUnCheck.png);
}

QRadioButton::indicator:checked 
{
	image: url(:/image/images2/icon_single_radioCheck.png);
}</string>
          </property>
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btnSelectMenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>43</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton { /* all types of tool button */
  text-align: left;
  border: 1px solid #DFE3E5;
  border-radius: 4px;
  padding-left: 10px;
  
font-size: 14px;
font-weight: 350;
color: #2D3440;
}

QPushButton::menu-indicator{
  bottom: 12px; right: 10px;
  image: url(:/image/images2/icon_single_pull_normal.png);
}
QPushButton::menu-indicator:hover{
 bottom: 12px; right: 10px;
 image: url(:/image/images2/icon_pull_hover.png);
}
QPushButton::menu-indicator:pressed {
  image: url(:/image/images2/icon_pull_press.png);
}
QPushButton::drop-down{
	border:none;
}
QPushButton::menu-button {
  text-align: left;
  border:none;
  /* 16px width + 4px for border = 20px allocated above */
  subcontrol-origin: padding;
  subcontrol-position: center right;
  width: 30px;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSelectedUpgrade">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
color: #868EB8;
font-size: 13px;
font-weight: 350;</string>
            </property>
            <property name="text">
             <string>对应升级的主控板：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textSelectedUpgrade">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
color: #868EB8;
font-size: 13px;
font-weight: 350;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignLeft|Qt::AlignTop">
         <widget class="QRadioButton" name="btnRadioImport">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>43</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton
{
	spacing: 6px;
	border:none;
color: #2D3440;
font-size: 16px;
font-weight: 350;
}
QRadioButton::indicator
{
	image: url(:/image/images2/icon_single_radioUnCheck.png);
}
QRadioButton::indicator:unchecked 
{
	image: url(:/image/images2/icon_single_radioUnCheck.png);
}

QRadioButton::indicator:checked 
{
	image: url(:/image/images2/icon_single_radioCheck.png);
}</string>
          </property>
          <property name="text">
           <string>导入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineImportedFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>43</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color: #2D3440;
padding-left: 12px;
padding-right: 12px;

font-size: 14px;
font-weight: 350;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnImportDir">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>43</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 14px;
font-weight: 350;
color: #2D3440;
padding-left:12px;
padding-right:12px;
border: 1px solid #DFE3E5;</string>
          </property>
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>44</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 3px;
background: #FAFDFF;
border: 1px solid #DFE3E5;
font-size: 16px;
font-weight: normal;
color: #2D3440;</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSingleUpgrade">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>44</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
  width: 196px;
  height: 44px;
  background: #0047BB;
  border-radius: 4px;
  color: #FAFDFF;
}
QPushButton::disabled{
  background: rgb(55, 71, 95);
}</string>
          </property>
          <property name="text">
           <string>升级</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
